solutions/dmoj-phidias.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: dmoj-phidias
```

Comment:
Source should be changed to IOI 2004

Reply:
Also why is this problem even in the guide. It's not that nice

Reply:
Yeah... Why is it still here?

Reply:
The problem ID and source in the contents folder is still "dmoj-phidias" and "dmoj" though. Do you want me to change those as well ?

Reply:
Yes please

Reply:
Or rather just remove the problem (I don't like it)